prow/plugins/trigger/generic-comment_test.go

```diff
@@ -1405,7 +1406,7 @@ func TestCommandFilter(t *testing.T) {
 			if err := config.SetPresubmitRegexes(testCase.presubmits); err != nil {
```

Comment:
We should move the tests over as well, please

Reply:
done.